scss/_variables.scss

```diff
@@ -363,29 +365,61 @@ $modal-lg: 720px !default;
 $modal-md: 540px !default;
 $modal-sm: 380px !default;
 
+//nav
+$nav-link-padding-y: .25rem !default;
+$nav-link-padding-x: .75rem !default;
+
 //navbar
-$navbar-padding-y: .5rem !default;
 $navbar-height: 3.5rem !default;
-$navbar-border-color: $border-color !default;
+$navbar-padding-y: .25rem !default;
+
+$navbar-light-color: rgba($dark, $text-muted-opacity) !default;
+$navbar-light-brand-color: inherit !default;
+$navbar-light-active-color: inherit !default;
+$navbar-light-disabled-color: rgba($dark, .3) !default;
+
+$navbar-dark-color: rgba($white, $text-muted-opacity) !default;
+$navbar-dark-brand-color: inherit !default;
+$navbar-dark-active-color: inherit !default;
+$navbar-light-disabled-color: rgba($white, .3) !default;
+
+
+$navbar-brand-padding-y: $nav-link-padding-y !default;
+$navbar-brand-image-height: 2rem !default;
+$navbar-brand-margin-right: 0 !default;
+$navbar-brand-font-size: $h3-font-size !default;
+$navbar-brand-font-weight: 600 !default;
 
-$navbar-toggler-padding-y: .75rem !default;
-$navbar-toggler-padding-x: .75rem !default;
-$navbar-toggler-margin: -.75rem !default;
 $navbar-toggler-font-size: 1rem !default;
-$navbar-toggler-size: 1.5rem !default;
-$navbar-toggler-item-height: 2px !default;
-$navbar-toggler-item-spacing: 7px !default;
-$navbar-toggler-transform-time: .2s !default;
-
-//sidenav
-$navbar-vertical-width: 15.625rem !default;
-$navbar-vertical-narrow-width: 4rem !default;
-$navbar-vertical-dark-bg: $dark !default;
-$navbar-vertical-dark-color: $white !default;
-$navbar-vertical-padding: 1rem !default;
-$navbar-vertical-active-accent-color: $primary !default;
-
-$navbar-nav-link-padding-x: .75rem !default;
+$navbar-toggler-padding-x: 0 !default;
+$navbar-toggler-padding-y: 0 !default;
+$navbar-light-toggler-icon-bg: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" class="icon icon-tabler icon-tabler-align-justified" width="24" height="24" viewBox="0 0 24 24" stroke-width="2" stroke="' + $body-color + '" fill="none" stroke-linecap="round" stroke-linejoin="round"><path stroke="none" d="M0 0h24v24H0z"/><line x1="4" y1="6" x2="20" y2="6" /><line x1="4" y1="12" x2="20" y2="12" /><line x1="4" y1="18" x2="20" y2="18" /></svg>') !default;
```

Comment:
24px != 1rem and it's not encoded string

Reply:
@codecalm
~~~
$form-icon-size: calculatePx($form-check-input-width) - $form-check-input-border * 2 !default;
$form-check-input-checked-bg-image: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' width='{$form-icon-size}' height='{$form-icon-size}' viewBox='0 0 {$form-icon-size} {$form-icon-size}'><path fill='none' stroke='#{$form-check-input-checked-color}' stroke-linecap='round' stroke-linejoin='round' stroke-width='3' d='M4 8.5L6.5 11l6-6' transform='scale(.8)' transform-origin='50% 50%'/></svg>") !default;
$form-check-input-checked-bg-image-dark: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' width='{$form-icon-size}' height='{$form-icon-size}' viewBox='0 0 {$form-icon-size} {$form-icon-size}'><path fill='none' stroke='#{$dark}' stroke-linecap='round' stroke-linejoin='round' stroke-width='3' d='M4 8.5L6.5 11l6-6' transform='scale(.8)' transform-origin='50% 50%'/></svg>") !default;
$form-check-radio-checked-bg-image: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' width='{$form-icon-size}' height='{$form-icon-size}' viewBox='0 0 {$form-icon-size} {$form-icon-size}'><circle r='3' fill='#{$form-check-input-checked-color}' cx='8' cy='8' /></svg>") !default;
~~~